apps/devu-api/utils/use-llm-provider.ts

```
@@ -0,0 +1,92 @@
+        apiKey: options.credentials.apiKey,
+        headers: options.configuration?.headers,
+      })
+
+    case 'mistral':
+      return createMistral({
+        apiKey: options.credentials.apiKey,
+        headers: options.configuration?.headers,
+      })
+
+    case 'openai':
+      const isOpenRouter = options.configuration?.baseUrl === 'https://openrouter.ai/api/v1'
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)